src/main/resources/assets/tconstruct/book/materials_and_you/en_us/modifiers/netherite.json

```diff
@@ -0,0 +1,14 @@
+{
+  "modifier_id": "tconstruct:netherite",
+  "text": [
+    {
+      "text": "The strength of the nether flows intensely through your tool."
+    }
+  ],
+  "effects": [
+    "Strengthens the tool, makes it immune to external damage such as fire",
```

Comment:
That sounds like it dos the same thing as worldbound. This does not just make it immune to external damage, it also boosts durability, attack damage, and mining speed.